configs/components/fesom/fesom-2.0.yaml

```diff
@@ -203,14 +216,40 @@ outdata_targets:
 
 choose_use_icebergs:
         True:
-            ib_num_old: 0
-            ib_num_new: 0
-
-            add_restart_in_files:
-                    icb_restart: icb_restart
-
-            add_restart_out_files:
-                    icb_restart: icb_restart
```

Comment:
Was there a reason why this was moved out from here and placed in the `restart_in_files` out of the `choose_use_icebergs`? Like it did not work otherwise?